browser/brave_content_browser_client.cc

```diff
@@ -138,6 +138,21 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #include "brave/browser/ftx/ftx_protocol_handler.h"
 #endif
 
+#if BUILDFLAG(BRAVE_ADS_ENABLED)
+#include "brave/browser/brave_ads/ads_service_factory.h"
+#include "brave/browser/brave_rewards/rewards_service_factory.h"
```

Comment:
rewards needs to be in a rewards buildflag, not ads

Reply:
Fixed, buildflag was removed here: https://github.com/brave/brave-core/pull/9769